osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1551,6 +1551,39 @@
         "OSMO": 725
       }
     },
+    {
+      "description": "The native EVM, governance and staking token of the Acrechain",
+      "denom_units": [
+        {
+          "denom": "ibc/BB936517F7E5D77A63E0ADB05217A6608B0C4CF8FBA7EA2F4BAE4107A7238F06",
+          "exponent": 0,
+          "aliases": ["aacre"]
+        },
+        {
+          "denom": "acre",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/BB936517F7E5D77A63E0ADB05217A6608B0C4CF8FBA7EA2F4BAE4107A7238F06",
+      "name": "Acre",
+      "display": "acre",
+      "symbol": "ACRE",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-490",
+        "source_denom": "aacre"
+      },
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/acrechain/images/acre.svg",
+	      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/acrechain/images/acre.png"
+      },
+      "coingecko_id": "arable-protocol",
+      "keywords": [
+        "osmosis-main",
```

Comment:
this asset has not yet been approved by Osmosis Governance for Osmosis-main. Please remove this keyword.

Reply:
Removed osmosis-main keyword. https://github.com/osmosis-labs/assetlists/pull/289/files#diff-9cae35db9ea3106b3271726ae08f09feeafa27a1d6978836b483d739a2252f53R2040